src/tv2-common/updatePolicies/shouldRemoveOrphanedPartInstance.ts

```diff
@@ -1,9 +0,0 @@
-import { BlueprintRemoveOrphanedPartInstance, IRundownUserContext } from '@tv2media/blueprints-integration'
-import { PartMetaData } from 'tv2-common'
-
-export function shouldRemoveOrphanedPartInstance(
```

Comment:
I might be missing something, but I don't see this logic being reimplemented in `syncIngestUpdateToPartInstance` using the new `removePartInstance` approach. Is there a task for that?

Reply:
I don't think there is a task for that.
I wasn't sure if this was something that we still used.

Is there a way that I can test that we have the correct behaviour so that I can verify that it still works or reimplement it?

Reply:
You'll find some repro steps and details about the original issue in SOF-664.